### ISSUE TITLE
[Docs] Some packages are private, thus not installable

### ISSUE THREAD

**Issue by ffoodd** [sonarwhal/sonarwhal#1069]
Some subpackages are private, thus not installable, and yet publicly documented (eg: [rule-webpack-config](https://github.com/sonarwhal/sonarwhal/blob/d80f6a26218233eed9b4dec98febc1deacb2d372/packages/rule-webpack-config/package.json#L51) or [rule-minified-js](https://github.com/sonarwhal/sonarwhal/blob/d80f6a26218233eed9b4dec98febc1deacb2d372/packages/rule-minified-js/package.json#L50)).

This is highly deceptive. :/

I suppose those packages are WIP and documentation is done along, but a simple message on top of documentation pages and/or removing the `npm install` part would tell people that they aren't installable for now.

Or did I missed something?

**Comment by ffoodd** [comment]
Wow (∩✿ᴥ✿)⊃━☆ﾟ.*

So they should be included in the next release?

This project is awesome!

**Comment by ffoodd** [comment]
Hi there, I got questions :)

When trying to use minified-js rules, following the doc doesn't work: adding `js-minifiy-check` in `.sonarwhal.rc` tries to install `rule-js-minify-check`; but adding `minified-js` doesn't throw any error when JS isn't minified.

Am I still missing something?


On another hand, is there any reason not to provide a `bin` value in your main package? This would make working with sonarwhal as a devDep easier, wouldn't it?

**Comment by molant** [comment]
Can you share the js you are analyzing? The threshold we are using is 75 so maybe we need to tweak it.

Sonarwhal should have a bin. What are you trying to do that doesn't work?

**Comment by kshyju** [comment]
@ffoodd Sorry! The documentation had wrong rule name. It should be `minified-js`, not `minifiy-check`! 

`minifiy-check` was the old name of the rule and i missed to update the readme file when i did the renaming ( **_find_** option in vs code missed it 😭)

This should work

```
{
    "connector": {...},
    "formatters": [...],
    "parsers": ["javascript"],
    "rules": {
        "minified-js": "error"
    },
    ...
}

```

I opened a [PR](https://github.com/sonarwhal/sonarwhal/pull/1076) to fix the readme file. Thanks!

**Comment by ffoodd** [comment]
@molant Forget about my bin sentence, I used a wrong path -_-
@kshyju Thanks, I guessed there was some kind of mistake. I managed to get this working \o/

Thanks again, you rock!

**Comment by molant** [comment]
> you rock!

Thanks!
